core/storage/app_feedback_report/gae_models_test.py

```diff
@@ -333,7 +346,8 @@ class InvalidFilter(enum.Enum):
                     InvalidFilter.INVALID_FIELD) # type: ignore[arg-type]
 
     def _mock_query_filters_returns_empy_list(
-            self, projection: bool, distinct: bool) -> List[Any]: # pylint: disable=unused-argument
+        self, projection: bool, distinct: bool  # pylint: disable=unused-argument
+    ) -> List[Any]:
```

Comment:
Can the return value not include Any?

Reply:
I didn't find proof of any other type so I changed this to List[str].